exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -34,13 +36,15 @@ const maxBatchByteSize = 3000000
 // prwExporter converts OTLP metrics to Prometheus remote write TimeSeries and sends them to a remote endpoint.
 type prwExporter struct {
 	endpointURL     *url.URL
-	client          *http.Client
+	client          *retryablehttp.Client
```

Comment:
Sorry for keep things changing, but I think we will not be able to use go-retryablehttp.

1. It does not have jitter on the exponential backoff.
2. It does not have the same configuration parameters that are avialble in `exporterhelper.RetrySettings`. (In theory you could create a conversion but I don't think it is worth).

So instead of using the this client, I suggest that instead you use https://pkg.go.dev/github.com/cenkalti/backoff/v4#Retry

The idea is that you would setup an [ExponentialBackoff](https://pkg.go.dev/github.com/cenkalti/backoff/v4#ExponentialBackOff) 
 with the exactly the same parameters of `exporterhelper.RetrySettings`  and would wrap the `execute` method in a [Retry](https://pkg.go.dev/github.com/cenkalti/backoff/v4#Retry)